### PR TITLE
[#379] PWM switch fix

### PR DESCRIPTION
**Pull request by ojasgulati**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [x] Bug fix
- [ ] Feature implementation
- [ ] Doc updates

* **What changes have you introduced?**

 PWM switch fix

* **Does this PR introduce a breaking change?**
NO

### CODE REVIEW

**Comment by ojasgulati** [comment]
@AakashMallik

File: src/screen/WaveGenerator/WaveGenerator.js [hunk 1]
```diff
@@ -112,8 +109,8 @@ class WaveGenerator extends Component {
     isConnected &&
       loadBalancer.sendData(ipcRenderer, 'linker', {
         command: 'SET_CONFIG_WAV_GEN',
-        wave: activePreview.wave,
-        digital: activePreview.digital,
+        wave,
+        digital,
```
Comment (AakashMallik) [inline src/screen/WaveGenerator/WaveGenerator.js]: You don't need to do this now.... you can directly unroll the state here... 
```
{
...this.state
}
```

File: src/screen/WaveGenerator/components/SquareWaveParameters.js [hunk 2]
```diff
@@ -107,11 +107,11 @@ const SquareWaveParameters = ({
         step={1}
         minTitleWidth="60px"
         minUnitWidth="40px"
-        disabled={!activePreview.digital}
+        disabled={!digital}
       />
     </OptionsRowWrapper>
     <Divider />
-    <OptionsRowWrapper></OptionsRowWrapper>
+    <OptionsRowWrapper />
```
Comment (AakashMallik) [inline src/screen/WaveGenerator/components/SquareWaveParameters.js]: Remove this... this Option Wrapper is not wrapping anything.. .redundant code

**Comment by ojasgulati** [comment]
@AakashMallik Done

**Pull request merged**